.github/workflows/main.yml

```diff
@@ -27,6 +27,8 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Black Validation
         uses: psf/black@stable
+        with:
+          version: "22.8.0"  # Last version which can be used in py3.6 and py3.7
```

Comment:
are we supporting py3.6 anymore?
can we use a newer version which supports py3.7? (23.3.0)

Reply:
22.8.0 is the last version for 3.6 and 3.7
After that they only have 3.8 support